docs/libcurl/opts/CURLOPT_V4_SIGNATURE.3

```diff
@@ -0,0 +1,65 @@
+.\" **************************************************************************
+.\" *                                  _   _ ____  _
+.\" *  Project                     ___| | | |  _ \| |
+.\" *                             / __| | | | |_) | |
+.\" *                            | (__| |_| |  _ <| |___
+.\" *                             \___|\___/|_| \_\_____|
+.\" *
+.\" * Copyright (C) 1998 - 2020, Daniel Stenberg, <daniel@haxx.se>, et al.
+.\" *
+.\" * This software is licensed as described in the file COPYING, which
+.\" * you should have received as part of this distribution. The terms
+.\" * are also available at https://curl.haxx.se/docs/copyright.html.
+.\" *
+.\" * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+.\" * copies of the Software, and permit persons to whom the Software is
+.\" * furnished to do so, under the terms of the COPYING file.
+.\" *
+.\" * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+.\" * KIND, either express or implied.
+.\" *
+.\" **************************************************************************
+.\"
+.TH CURLOPT_V4_SIGNATURE 3 "03 Jun 2020" "libcurl 7.72.0" "curl_easy_setopt options"
+.SH NAME
+CURLOPT_V4_SIGNATURE \- V4 signature (used by aws/outscale))
+.SH SYNOPSIS
+.nf
+#include <curl/curl.h>
+
+CURLcode curl_easy_setopt(CURL *handle, CURLOPT_V4_SIGNATURE,
+                          char *providers_infos);
+.SH DESCRIPTION
+performe v4 signature as decribe here: https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html
+or here https://wiki.outscale.net/display/EN/About+Signatures+of+API+Requests
+pass a string that contain a provider prefix that will be used to forge the signature.
```

Comment:
Please describe this as detailed as possible without adding URLs. URLs should be avoided in the man page, but if they really add a value they could be added below somewhere perhaps with its own subtitle.